mistralrs-core/src/pipeline/speculative.rs

```diff
@@ -30,27 +31,28 @@ pub struct SpeculativeLoader {
 }
 
 impl Loader for SpeculativeLoader {
-    fn load_model(
+    #[allow(clippy::type_complexity, clippy::too_many_arguments)]
+    fn load_model_from_hf(
```

Comment:
Like mentioned below, can we document that this falls back to searching locally if it cannot find the files on HF hub?

Reply:
Added to `Loader` trait.